02 ReactApp/src/pods/organization/component.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class OrganizationComponent extends Component {
+
+  componentDidMount() {
+    this.props.fetchMembers();
+  }
+
+  render() {    
+    return (
```

Comment:
You could do `this.props.members.map(this._renderItem):`
```
_renderItem = (member) => {
    <li key={member.id}>
        <span>{member.login}</span>
    </li>
}
```

Or 
```
this.props.members.map((member) => <MemberListItem member={member}/>):
```
```
const MemberListItem = ({member}) => {
    <li key={member.id}>
        <span>{member.login}</span>
    </li>
}
```

With the second approach - I could easily wrap with `pure`. If logic needed it.